app/assets/stylesheets/mountain_view/styleguide.css

```diff
@@ -18,6 +18,12 @@ body{
   background: #777;
   padding: 20px;
 }
+
+.mv--black-background{
```

Comment:
Do we need this here?
Seems not to be used but by the examples. If so, that's something the user should provide as part of their customization, as we try to keep the styles to only the ones used by the default views.

Reply:
Good catch, I'd put it in for the screen shot and forgot to take it out... oops!